src/test/java/org/mule/tools/api/repository/ArtifactInstallerTest.java

```diff
@@ -67,69 +63,63 @@
 
   @Before
   public void before() throws IOException {
     logMock = mock(Log.class);
     handler = new DefaultArtifactHandler(TYPE);
-    List<ArtifactRepository> remoteArtifactRepositories = new ArrayList<>();
-    aetherRepositorySystemMock = mock(RepositorySystem.class);
-    aetherRepositorySystemSessionMock = mock(RepositorySystemSession.class);
-    installer = new ArtifactInstaller(logMock, remoteArtifactRepositories, aetherRepositorySystemMock,
-                                      aetherRepositorySystemSessionMock);
+    installer = new ArtifactInstaller(logMock);
     outputFolder.create();
     artifactFileFolder.create();
     artifact = new DefaultArtifact(GROUP_ID, ARTIFACT_ID, VERSION, SCOPE, TYPE, CLASSIFIER, handler);
   }
 
-  @Test
-  public void getNormalizedVersionFromBaseVersionTest() {
-    artifact = new DefaultArtifact(GROUP_ID, ARTIFACT_ID, VERSION + "-" + TIMESTAMP, SCOPE, TYPE, CLASSIFIER, handler);
-    String actual = installer.getNormalizedVersion(artifact);
-    assertThat("The normalized version is different from the expected", actual, equalTo(VERSION + "-" + SNAPSHOT_VERSION));
-  }
-
-
-  @Test
-  public void getNormalizedVersionFromVersionTest() {
-    String expected = installer.getNormalizedVersion(artifact);
-    assertThat("The normalized version is different from the expected", expected, equalTo(VERSION));
-  }
-
-  @Test
-  public void getFormattedOutputDirectoryTest() {
-    File actual = ArtifactInstaller.getFormattedOutputDirectory(outputFolder.getRoot(), artifact);
-    File expected = new File(outputFolder.getRoot(),
-                             OUTPUT_DIRECTORY);
-    assertThat("Actual formatted output directory is not the expected", actual.getAbsolutePath(),
-               equalTo(expected.getAbsolutePath()));
-  }
-
-  @Test
-  public void getFormattedFileNameTest() {
-    String actual = installer.getFormattedFileName(artifact);
-    String expected = ARTIFACT_ID + "-" + VERSION + "-" + CLASSIFIER + "." + TYPE;
-    assertThat("Formatted file name is different from the expected", actual, equalTo(expected));
-  }
-
-  @Test
-  public void getFormattedFileNameWithoutClassifierTest() {
-    artifact = new DefaultArtifact(GROUP_ID, ARTIFACT_ID, VERSION, SCOPE, TYPE, null, handler);
-    String actual = installer.getFormattedFileName(artifact);
-    String expected = ARTIFACT_ID + "-" + VERSION + "." + TYPE;
-    assertThat("Formatted file name is different from the expected", actual, equalTo(expected));
-  }
-
-  @Test
-  public void getFormattedFileNameWithoutHandlerTest() {
-    artifact = new DefaultArtifact(GROUP_ID, ARTIFACT_ID, VERSION, SCOPE, TYPE, CLASSIFIER, null);
-    String actual = installer.getFormattedFileName(artifact);
-    String expected = ARTIFACT_ID + "-" + VERSION + "-" + CLASSIFIER + "." + TYPE;
-    assertThat("Formatted file name is different from the expected", actual, equalTo(expected));
-  }
+  // @Test
```

Comment:
Why did we stop testing this?
Is there a test for DefaultMavenRepositoryLayoutUtils, if not please create it and move those tests there